comixed-tasks/src/test/resources/application.properties

```diff
@@ -20,7 +20,7 @@ spring.jackson.deserialization.fail-on-unknown-properties=false
 
 # Logging
 logging.level.root=OFF
-logging.level.org.comixed=INFO
+logging.level.org.comixed=DEBUG
```

Comment:
Is pushing this change is voluntary? (in production debug level can impact performance)

Reply:
This is only used as a part of the JUnit test run: the file's in ```src/test/resources```.

Reply:
I didn't see it was for the tests :)